src/main/java/Car.java

```diff
@@ -0,0 +1,5 @@
+public class Car extends Transport {
+    public Car(String typeOfTransport) {
+        super(typeOfTransport);
+    }
```

Comment:
```suggestion
    public Car() {
        super("автобус");
    }
```
Тут можно сразу вот так написать.
И аналогично в других транспортных средствах.